### DISCUSSION TITLE
Push more information to discord

### DISCUSSION THREAD

**Discussion by jswigart** [pascal-zarrad/tc-discord-webhooks#34] [Ideas]
I'd like to request some configuration options to push more data to discord, such as

- Agent that is building the build
- Hyperlinked build Id that can take you to the build in team city
- A customizable discord payload would even be great, with access to build variables similarly to build steps(if this is even possible), this way the user can have a lot more expressiveness with what is displayed,

The Jenkins discord notifier is a good example of hyperlink usage to provide e

**Comment by pascal-zarrad** [comment]
Hi!

agent ID and a hyperlink to the agent that is doing the build could be added. It is a quite easy addition. 
This data would be always shown.
Currently I do not have much time, but maybe I can add it in the near future.

Completly making the notifications customizeable would be great and was planned in the past. But due to limited time, I cannot implement it. Making the notifications customizeable will take a huge amount of work.

I reopened the issue #11, maybe someone would like to contribute the feature.
